examples/playground/src/deck-json-converter.js

```diff
@@ -11,28 +12,20 @@
 // * Optionally, error checking could be applied, but ideally should leverage
 //   non-JSON specific mechanisms like prop types.
 
-import {_JSONConverter, _shallowEqualObjects, _parseExpressionString} from '@deck.gl/json';
+import {_shallowEqualObjects, _parseExpressionString} from '@deck.gl/json';
+import {MapView, FirstPersonView, OrbitView, OrthographicView} from '@deck.gl/core';
+import {COORDINATE_SYSTEM} from '@deck.gl/core';
+import GL from '@luma.gl/constants';
 import enhancedFetch from './enhanced-fetch';
 
-import DeckJSONConfiguration from './deck-json-configuration';
-
-export const DEFAULT_MAP_PROPS = {style: 'mapbox://styles/mapbox/light-v9'};
-
-export default class DeckJSONConverter extends _JSONConverter {
-  constructor(props) {
-    super({
-      ...props,
-      // Inject default deck configuration
-      configuration: new DeckJSONConfiguration(props.configuration)
-    });
-    this.configuration.preProcessClassProps = this.preProcessClassProps.bind(this);
-  }
-
-  preProcessClassProps(Class, props) {
+export const DECK_JSON_CONVERTER_CONFIGURATION = {
+  // Support all `@deck.gl/core` Views by default
+  classes: {MapView, FirstPersonView, OrbitView, OrthographicView},
+  enumerations: {COORDINATE_SYSTEM, GL},
+  preProcessClassProps(Class, props, configuration) {
     props.fetch = props.fetch || enhancedFetch;
-    return getJSONLayerProps(Class, props, this.configuration);
-  }
-
+    return convertFunctions(Class, props, configuration);
+  },
   postProcessConvertedJson(json) {
```

Comment:
I don't like this secret key swap. Why is this needed? Can we handle the view state update in the playground app instead?